css/style.css

```diff
@@ -6,3 +6,228 @@
+     background-image: url("https://images.pexels.com/photos/1879877/pexels-photo-1879877.jpeg?auto=compress&cs=tinysrgb&w=600");
+     background-position: cover;
+     background-repeat: no-repeat;
+     color: whitesmoke;
+     height: 700px;
+     width: 50%;
+     padding-bottom: 50px;
+}
+
+ .h3 {
+     text-decoration: underline;
+}
+
+.h1 {
+     font-size: 30px;
+ }
+
+.date {
+     font-weight: bolder;
+}
+
+/* General Styles - Top Container*/
+
+.top-container {
+    padding: 5px;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    margin-bottom: 0;
+    
+}
+ 
+.logo {
+     display: flex;
+     flex-direction: row;
+     font-size: 80px;
+     font-weight: bold;
+     justify-content: center;
+     padding-right: 68px;
+     text-decoration: underline;
+}
+
+.top-right {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+ }
+
+ .top-right-p1 {
+     padding: 5px 0 0 100px;
+     font-weight: 600;
+}
+
+.top-right-p2 {
+    padding: 5px 0 0 10px;
+    font-weight: 600;
+}
+
+.hburger-img {
+    width: 60px;
+    height: 60px;
+}
+
+h4 {
+    color: rgb(88, 85, 85);
+}
+
+/* General Styles - Main container and aside container */
+.main-container {
+    display: flex;
+    flex-direction: row;
+    height: 800px;
+    margin: 50px 0 10px 0;
+}
+
+.aside-container {
+    float: right;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;   
+    padding-left: 100px;
+}
+
+.aside-border {
+    border: 1px solid rgb(180, 175, 175);
+    margin-bottom: 40px;
+    padding: 15px;
+    font-weight: bolder;  
+}
+
+.stories {
+    color: black;
+    font-weight: 600;
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: row;
+    margin: 15px 0 25px 0;
+    border-bottom: 1px solid black;
+    padding: 15px 0 25px 0;
+}
+
+/* Article Styles */
+
+.article-container {
+    display: flex;
+    flex-direction: row;
+    border: 2px solid rgb(204, 200, 200);
+    border-radius: 4px;
+    padding: 20px;
+}
+
+article:first-child {
+    display: flex;
+    flex-direction: column;
+    flex: 0;
+    order: 2;
+    background-color: rgb(196, 207, 203);
+    height: 700px;
+    border-radius: 20px;
+    padding: 25px;
+    margin-right: 10px;
+}
+
+.article2 {
+    display: flex;
+    flex-direction: column;
+    flex: 2;
+    order: 1;
+    padding: 25px;
+}
+
+.article3 {
+    display: flex;
+    flex: 2;
+    flex-direction: column;
+    order: 3;
+    padding: 25px; 
+}
+
+.img-1 {
+     max-width: 300px;
+     max-height: 100%;
+ }
+
+
+.img-2 {
+     max-width: 250px;
+     max-height: 100%;
+ }
+
+.img-3 {
+     max-width: 300px;
+     max-height: 100%;
+
+ }
+
+.link {
+     color: whitesmoke;
+     text-decoration: none;
+ }
+
+ button {
+     height: 40px;
+     width: 100px;
+     background-color: black;
+     color: white;
+     margin: 8px 0 0 10px;
+ }
+
+/* Footer styles*/
+
+footer {
+    left: 0;
+    bottom: 0;
```

Comment:
These are not doing anything here. The footer is looking offset to the right because you are setting `width: 100%;` which the browser takes and then adds the padding to it. if you remove width the footer will naturaly fill the space